SmartDeviceLink/SDLChoiceSetManager.m

```diff
@@ -459,18 +513,79 @@ - (SDLKeyboardProperties *)sdl_defaultKeyboardConfiguration {
 
 #pragma mark - Getters
 
+- (NSMutableSet<SDLChoiceCell *> *)preloadedMutableChoices {
+    __block NSMutableSet<SDLChoiceCell *> *set = nil;
+    [self sdl_runSyncOnQueue:^{
+        set = self->_preloadedMutableChoices;
+    }];
+
+    return set;
+}
+
+- (NSMutableSet<SDLChoiceCell *> *)pendingMutablePreloadChoices {
+    __block NSMutableSet<SDLChoiceCell *> *set = nil;
+    [self sdl_runSyncOnQueue:^{
+        set = self->_pendingMutablePreloadChoices;
+    }];
+
+    return set;
+}
```

Comment:
I'm not sure that this is necessary. Every place but one that access it is already synchronized. That might be a better approach.

Reply:
Removed and added synchronization to read access to `pendingMutablePreloadChoices` and `preloadedMutableChoices`.